### PR TITLE
API CALL MADE AND APP WORKING SUCCESSFULLY 

### PR DESCRIPTION
**Pull request by harshagl2002**

### CODE REVIEW

File: Project/Travel App/Travel App/ContentView.swift [hunk 1]
```diff
@@ -5,31 +5,7 @@ struct ContentView: View {
...
-                    .navigationBarBackButtonHidden(true)
-                    .onAppear() {
-                        NotificationCenter.default.addObserver(forName: NSNotification.Name("status"), object: nil, queue: .main) { (_) in
-                            
-                            self.status = UserDefaults.standard.value(forKey: "status") as? Bool ?? false
-                        }
-                    }
-                }
-            }
-        }
```
Comment (MattGeimer) [inline Project/Travel App/Travel App/ContentView.swift]: @harshagl2002 This needs to be changed back (this was some of the rest of the group's code)

File: Project/Travel App/Travel App/FlightSearch/Information.swift [hunk 2]
```diff
@@ -0,0 +1,10 @@
+//
+//  Information.swift
+//  FlighSearch
+//
+//  Created by Harsh Agarwal on 20/03/21.
+//
+
+import Foundation
+ 
+
```
Comment (MattGeimer) [inline Project/Travel App/Travel App/FlightSearch/Information.swift]: Is this file empty? If so, it shouldn't be in the repository

**Review by MattGeimer — CHANGES_REQUESTED** [review]
Needs a couple of small changes (see comments) otherwise looks good!

**Comment by harshagl2002** [comment]
Issues fixed

**Review by MattGeimer — APPROVED** [review]

**Review by rachwei — APPROVED** [review]
Looks good-- nice job!

**Pull request merged**